scripts/mk_all.lean

```diff
@@ -68,7 +68,9 @@ def mkAllCLI (args : Parsed) : IO UInt32 := do
       updates := updates + 1
   if updates == 0 then
     IO.println "No update necessary"
-  return updates
+  -- Make sure to return an exit code of at most 125, so this return value can be used further
+  -- in shell scripts.
+  return min updates 125
```

Comment:
```suggestion
  return min (numberErrorFiles % 128) 125
```
See above for a justification.

Reply:
Changed this similarly; let's keep the discussion in the other comment.